Birthday-2024-Project/Scripts/GridLogic.gd

```diff
@@ -1,89 +1,146 @@
 extends TileMap
 class_name GridLogic
 
-const MAX_HEIGHT : int = 20
-const MAX_WIDTH : int = 20
+const MAX_HEIGHT : int = 4 #doubled for negative
```

Comment:
could we make grid bigger? I recall a discussion on discord that we have a lot of big shapes and bigger grid would help in utilizing them

Reply:
![image](https://github.com/Saplings-Projects/Birthday-2024/assets/9286524/196132da-6667-4330-af7f-9b14b88a8df9)

This was the consensus when I asked on the Discord. If you see the whole grid, the big Fauna piece isn't that bad. I suppose people won't be able to make anything too outlandish like a long diagonal line. But this could be expanded upon as a stretch goal if we have the time to implement a mobile camera system so you can move around the larger grid

Reply:
What about scaling the grid and pieces if more squares are to be used?
Also sorry, I should have known this...

Reply:
The issue is scaling or zooming the camera can cause the pieces' art to diminish in quality. Pixel-y if too small or blurry if too big